opencog/nlp/learn/README

```diff
@@ -141,30 +144,44 @@ So, set up the text-ingestion pipeline:
    (add-to-load-path "/usr/local/share/opencog/scm")
    (add-to-load-path ".")
 ```
-
-6) Select a language, and copy one of the `pair-count-??.scm` files
-   from the `run` directory to your working directory. Review and edit
-   the configuration as necessary. It contains the REPL port number,
-   and a default prompt: These are set up by default to avoid conflicts
-   and confusion, and to allow multiple languages to be processed at
-   the same time.  It also contains the database credentials -- that's
-   probably the only thing you need to change.
-
-7) Start the REPL server. Eventually, you can use the
-   `run-all-servers.sh` file in the `run` directory to do this;
+4) Copy the following files from the `opencog/opencog/nlp/learn/run` directory
+   to your working directory:
+
+   - run-server-parse.sh
+   - observe-launch.scm
+   - opencog-en.conf
+   - utilities.scm
+   - wiki-ss-en.sh
+   - ss-one.sh
+   - submit-one.pl
+   - nonbreaking_prefixes (the complete folder)
```

Comment:
is there a reason you are not copying the entire directory? There are various useful scripts in there, such as renice.sh that can make a big difference on a running system. Similar remarks for the shutdown scripts, which are needed to reboot containers/VM's/machines without corrupting the data in them.

Reply:
Just following your README instructions, where you first instruct to copy only some files. But no problem, changed to copy all directory to your workdir.